src/SignInPage/ResponsiveSocialButton.js

```diff
@@ -15,6 +16,12 @@ type Props = {
   children: string,
 };
 
+const clickOnSocialLoginTracker = (type: string) =>
```

Comment:
Why is this called `clickOnSocialLoginTracker`? Sounds as if it was clicking on something

Reply:
I wanted to use similar name as the event tracker name, but yes, you're right, removing the `clickOn` is better for the function name. fixed :)